lib/semantics/resolve-labels.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef FORTRAN_SEMANTICS_RESOLVE_LABELS_H_
+#define FORTRAN_SEMANTICS_RESOLVE_LABELS_H_
+
+namespace Fortran {
```

Comment:
C++17 allows `namespace Fortran::parser` and `namespace Fortran::semantics`, and it would be better if the code were consistent here with other usage like that instead of using nested `namespace`.